tests/suite/manifests/upstream-settings-policy/invalid-svc-usps.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: gateway.nginx.org/v1alpha1
+kind: UpstreamSettingsPolicy
+metadata:
+  name: latte-svc-usp
+spec:
+  zoneSize: 512k
+  targetRefs:
+  - group: core
+    kind: Service
+    name: latte
```

Comment:
let's call this `dne` or `does-not-exist` to make it clear what we are testing here